### PR TITLE
added esquire suffix

### PR DESCRIPTION
**Pull request by gruberjl**
I added the esquire suffix to resolve an issue with my code.

### CODE REVIEW

File: lib/parser.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var suffixes = [
   'PE', 'PHD', 'SJ', 'SR', 'V', 'VI', 'VP', 'V', 'I', 'VI', 'VII', 'VIII',
   'CNP', 'CPA', 'DDS', 'DMin', 'DMA ', 'DMus', 'OD', 'DO', 'PharmD', 'PhD',
   'PsyD', 'DVM', 'EI', 'EIT', 'Esq', 'JD', 'LLS', 'LP', 'LPN',
-  'MD', 'PE', 'RA', 'RLA', 'RLS', 'RN', 'SE'
+  'MD', 'PE', 'RA', 'RLA', 'RLS', 'RN', 'SE', 'Esquire', 'ESQUIRE'
```
Comment (mklaber) [inline lib/parser.js]: We're already using case-insensitive comparisons [here](https://github.com/mklaber/node-another-name-parser/blob/master/lib/util.js#L14). Can you remove one of these?

**Review by mklaber — CHANGES_REQUESTED** [review]

File: lib/parser.js (hunk 1 above)
Comment (mklaber) [reply lib/parser.js]: Also, can you write a test for this? (I know there's not a test for every suffix but it'd be nice to have a test for any new additions.)

**Review by gruberjl — COMMENTED** [review]
removed extra esquire

**Comment by mklaber** [comment]
@gruberjl not sure if you saw my other comment (that I thought of after my initial response): can you add a quick test for this?

**Review by mklaber — APPROVED** [review]
Looks great, thanks.

**Comment by mklaber** [comment]
Thanks @gruberjl for your contribution. I've published the updated [npm package](https://www.npmjs.com/package/another-name-parser).

**Pull request merged**